src/components/feedbacks/FeedbackItem.vue

```diff
@@ -1,5 +1,29 @@
 <template>
   <BaseBox>
+    <router-link :to="feedbackPageLink">
+      <li>
+        <article>
+          <p v-if="roadmap">{{ feedback.status }}</p>
+          <!-- status - activated in the Roadmap, hidden in the FeedbackList -->
+          <div class="feedback__votes">
+            <ArrowUp />
+            <p>{{ feedback.upvotes }}</p>
+          </div>
+          <div class="feedback__content">
+            <h2>{{ feedback.title }}</h2>
+            <p>{{ feedback.description }}</p>
+            <BaseButton :text="feedback.category" />
+          </div>
+          <div class="feedback__comments">
+            <IconComments />
+            <p>
+              {{ feedback.comments ? feedback.comments.length : 0 }}
```

Comment:
FYI you can also use here: feedback.comments.length || 0

Reply:
Maybe I do something wrong because it doesn't work when I try to implement your suggestion. I get  "[Vue warn]: Unhandled error during execution of render function" when the browser arrive to the feedback without a comment. My version works well. I merge this pr and keep your suggestion as a comment we can talk about it later ;)